packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -166,6 +167,10 @@ const VisitHeader: React.FC = () => {
     });
   }, []);
 
+  const { person } = useOmrsRestPatient(patient?.id);
+  const isDeceased = Boolean(person?.dead);
```

Comment:
You don't need this Boolean representation. The returned data type is already a boolean
So `const isDeceased = person?.dead` is enough


![image](https://user-images.githubusercontent.com/104269786/236411568-05d34b34-c541-40f5-b316-eff71cf2f017.png)